multigeiger/speaker.cpp

```diff
@@ -156,6 +163,33 @@ void tick_enable(bool enable) {
   }
 }
 
+void IRAM_ATTR alarm() {
+
+  int *sequence;
+
+  sequence = alarm_sequence;
+  // "high_pitch"
+  sequence[0] = 3000000;  // frequency_mHz
+  sequence[1] = 1;  // volume
+  sequence[2] = -1;  // LED, -1 = don't touch
+  sequence[3] = 400;  // duration_ms
+  // "low_Pitch"
+  sequence[4] = 1000000;
+  sequence[5] = 1;
+  sequence[6] = -1;
+  sequence[7] = 400;
+  // "off"
+  sequence[8] = 0;
+  sequence[9] = 0;
+  sequence[10] = -1;
+  sequence[11] = 0;  // END
+
+  // called from ISR!
+  portENTER_CRITICAL_ISR(&mux_audio);
+  isr_audio_sequence = sequence;
+  portEXIT_CRITICAL_ISR(&mux_audio);
```

Comment:
you are calling this from main loop, not from ISR.

Reply:
Comment removed